### PR TITLE
SITES-701 improve HTTP caching

### PR DESCRIPTION
**Pull request by freshtonic**

### CODE REVIEW

File: app/controllers/editorial/news_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class NewsController < EditorialController
 
     def index
       # TODO: return news editorial index
-      bustable_fresh_when(@sections)
+      with_caching(@sections)
```
Comment (maxious) [inline app/controllers/editorial/news_controller.rb]: Does this change the etag when news articles are added?

Comment (freshtonic) [inline app/controllers/editorial/news_controller.rb]: No, but from reading the code it does't render any news articles on the editorial news index so it shouldn't matter.

Comment (freshtonic) [inline app/controllers/editorial/news_controller.rb]: That particular invocation will only change the etag based on `@section.updated_at`.

**Comment by maxious** [comment]
LGTM + this seems to be working in a way that makes sense now

**Comment by ltankey** [comment]
LGTM

**Pull request merged**